1.20.0

matteobruni opened the pull request:
feat: responsive options
feat: resize keeps particles in place
feat: added overlap options to collisions
feat: added new bubbles preset (emitterAngled in demo)

matteobruni committed:
build: actual options in container, preparing for responsive options

ghost commented:
## DeepCode's analysis on [#7cc118](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/afec0f04db4cf4eed4b50463c92e98b72a3941a5/matteobruni/tsparticles/7cc118eb41e6a29fb370e0c8732fd671b0a9151c/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) found:

- :information_source: **1** minor issue. :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">'public' is implicit. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/7cc118eb41e6a29fb370e0c8732fd671b0a9151c/core/main/src/Options/Classes/Options.ts#L25">Options.ts:25</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/7cc118eb41e6a29fb370e0c8732fd671b0a9151c/core/main/src/Options/Classes/Options.ts#L34">Options.ts:34</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/7cc118eb41e6a29fb370e0c8732fd671b0a9151c/core/main/src/Options/Classes/Options.ts#L41">Options.ts:41</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/afec0f04db4cf4eed4b50463c92e98b72a3941a5/matteobruni/tsparticles/7cc118eb41e6a29fb370e0c8732fd671b0a9151c/pr/_/%2Fcore%2Fmain%2Fsrc%2FOptions%2FClasses%2FOptions.ts/javascript%2Ftslint%2Fmember-access/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/afec0f04db4cf4eed4b50463c92e98b72a3941a5/matteobruni/tsparticles/7cc118eb41e6a29fb370e0c8732fd671b0a9151c/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
feat: responsive options
feat: resize keeps particles in place

matteobruni committed:
feat: added overlap options to collisions
feat: added new bubbles preset (emitterAngled in demo)

matteobruni committed:
build: fixed editor

matteobruni committed:
build: update bubbles preset with prettier/eslint

matteobruni committed:
build: responsive demo added

matteobruni committed:
chore(release): published new version

 - ng-particles@2.3.0
 - inferno-particles@1.3.0
 - jquery-particles@1.20.0
 - preact-particles@1.20.0
 - react-tsparticles@1.20.0
 - svelte-particles@1.3.0
 - particles.vue@2.3.0
 - particles.vue3@1.3.0
 - tsparticles-editor@1.20.0
 - tsparticles@1.20.0
 - ng-particles-demo@1.5.0
 - editor-demo@1.5.0
 - inferno-particles-demo@1.3.0
 - jquery-particles-demo@1.5.0
 - tsparticles-demo@1.5.0
 - preact-particles-demo@1.5.0
 - react-tsparticles-demo@1.5.0
 - svelte-demo@1.3.0
 - particles.vue-demo@1.5.0
 - particles.vue3-demo@1.3.0
 - tsparticles-preset-60fps@1.5.0
 - tsparticles-preset-background-mask@1.5.0
 - tsparticles-preset-basic@1.5.0
 - tsparticles-preset-big-circles@1.5.0
 - tsparticles-preset-bouncing@1.5.0
 - tsparticles-preset-bubbles@1.5.0
 - tsparticles-preset-fire@1.5.0
 - tsparticles-preset-font-awesome@1.5.0
 - tsparticles-preset-fountain@1.5.0
 - tsparticles-preset-snow@1.5.0
 - tsparticles-preset-stars@1.5.0
 - tsparticles-shape-bubble@1.5.0
 - tsparticles-shape-heart@1.5.0
 - tsparticles-shape-multiline-text@1.5.0
 - tsparticles-shape-spiral@1.5.0
 - cra-template-particles-typescript@1.3.0
 - cra-template-particles@1.3.0
 - tsparticles-website@1.20.0
